mods/tools/hoes.lua

```diff
@@ -20,22 +20,16 @@ local function Hoe_Use(itemstack, user, pointed_thing)
 			end
 			
 			local soilness = minetest.get_item_group(node.name, "soil")
-			if soilness ~= 0 then
-				
+			local loamness = minetest.get_item_group(node.name, "loam")
+			if (soilness ~= 0 or loamness ~= 0) then
 				local param2 = 0
-				if user then
-					local user_pos = user:get_pos()
-					if user_pos then
-						param2 = minetest.dir_to_facedir(vector.subtract(pointed_thing.above, user_pos))
-					end
-				end
-				
-				minetest.swap_node(pos, {name = "overworld:dirt_tilled", param2=param2})
-				
+				minetest.swap_node(pos, {
```

Comment:
why did you get rid of the param2 selection for facing? That should not be removed.

Reply:
should be good

Reply:
Looks good now. But please remove the commented-out old version below, we always have that old version in git history.

Reply:
removed, if it all looks good feel free to squash and merge

Reply:
Looks good, thank you.